services/streammanager/kafka/kafkamanager_test.go

```diff
@@ -583,6 +597,30 @@ func TestSendBatchedMessage(t *testing.T) {
 		kafkaStats.publishTime = getMockedTimer(t, ctrl)
 		kafkaStats.prepareBatchTime = getMockedTimer(t, ctrl)
 
+		p := &pMockErr{error: nil}
+		pm := &ProducerManager{p: p}
+		sc, res, err := sendBatchedMessage(
+			context.Background(),
+			json.RawMessage(`[{"message":"ciao","userId":"123","topic":"some-topic"}]`),
+			pm,
+			"some-topic",
+		)
+		require.Equal(t, 200, sc)
+		require.Equal(t, "Kafka: Message delivered in batch", res)
+		require.Equal(t, "Kafka: Message delivered in batch", err)
+		require.Len(t, p.calls, 1)
+		require.Len(t, p.calls[0], 1)
+		require.Equal(t, []byte("123"), p.calls[0][0].Key)
+		require.Equal(t, []byte(`"ciao"`), p.calls[0][0].Value)
+		require.Equal(t, "some-topic", p.calls[0][0].Topic)
+		require.InDelta(t, time.Now().Unix(), p.calls[0][0].Timestamp.Unix(), 1)
+	})
+
+	t.Run("default-topic-test", func(t *testing.T) {
```

Comment:
Same as the other tests:

```suggestion
	t.Run("default topic test", func(t *testing.T) {
```

The missing characters are replaced by Go. Spaces are replaced with `_` but you don't have to remember that, just let it do its thing.

Reply:
This is addressed